drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -1310,7 +1309,7 @@ function DragAndDropBlock(runtime, element, configuration) {
         var items_in_zone_count = countItemsInZone(zone, [item_id.toString()]);
         if (configuration.max_items_per_zone && configuration.max_items_per_zone <= items_in_zone_count) {
             state.last_action_correct = false;
-            state.feedback = [{message: gettext("You cannot add any more items to this zone."), message_class: null}];
+            state.feedback = [{message: gettext("You cannot add any more items to this zone."), message_class: "limit"}];
```

Comment:
This is connected to my previous comment about this message class. `limit` is fine for internal use, but let's make this code a bit more readable by extracting it into an object property. We could do something like this at the top of the `DragAndDropBlock` declaration: `DragAndDropBlock.FEEDBACK_MESSAGE_CLASS_ZONE_LIMIT = 'limit';` (or something shorter, if it can be accurate; it's just an example).

Reply:
done